source/_patterns/05-components/form-item/form-item--checkboxes/form-item--checkboxes.twig

```diff
@@ -1,8 +1,9 @@
-<fieldset class="fieldset fieldset--checkboxes" aria-describedby="edit-checkboxes-description">
-  <legend class="fieldset__legend">
-    <span class="fieldset__legend-text">Checkboxes</span>
-  </legend>
-  <div class="fieldset__content">
+{% embed "@uswds/fieldset/fieldset.twig" with {
+  legend: 'Checkboxes',
+  description: '<p>The description for this form field.</p>',
+  description_id: 'edit-checkboxes-description'
+}%}
+  {% block fieldset_content %}
```

Comment:
Don’t use `usa-fieldset` for this. Don’t put a class on the fieldset and use `usa-label` for the legend class. We can also remove the `fieldset__content` wrapper.